src/Symfony/Component/HttpFoundation/Tests/CookieTest.php

```diff
@@ -262,6 +273,20 @@ public function testToString()
             ->withSameSite(null);
         $this->assertEquals($expected, (string) $cookie, '->__toString() returns string representation of a cleared cookie if value is NULL');
 
+        $expected = 'foo=deleted; expires='.gmdate('D, d M Y H:i:s T', $expire = time() - 31536001).'; Max-Age=0; path=/admin/; domain=.myfoodomain.com; secure; httponly; samesite=none; partitioned';
+        $cookie = Cookie::create('foo', null, 1, '/admin/', '.myfoodomain.com', true, true, false, 'none', true);
+        $this->assertEquals($expected, (string) $cookie, '->__toString() returns string representation of a cleared cookie if value is NULL');
+
+        $cookie = Cookie::create('foo')
+            ->withExpires(1)
+            ->withPath('/admin/')
+            ->withDomain('.myfoodomain.com')
+            ->withSecure(true)
+            ->withHttpOnly(true)
+            ->withSameSite('none')
+            ->withPartitioned(true);
+        $this->assertEquals($expected, (string) $cookie, '->__toString() returns string representation of a cleared cookie if value is NULL');
```

Comment:
```suggestion
        $this->assertSame($expected, (string) $cookie, '->__toString() returns string representation of a cleared cookie if value is NULL');
```
? could be changed for the others as well I guess

Reply:
Changed on all CookieTest. There are two tests `testConstructorWithDateTime` and `testConstructorWithDateTimeImmutable` with an ambigious assertion (integer value with string from DateTime). I will correct this in a dedicated commit.